tests/system/python/test_e2e_coap_PI.py

```diff
@@ -0,0 +1,111 @@
+# -*- coding: utf-8 -*-
+
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+
+""" Test system/python/test_e2e_coap_PI.py
+
+"""
+import os
+import subprocess
+import http.client
+import json
+import time
+import shutil
+import pytest
+
+
+__author__ = "Vaibhav Singhal"
+__copyright__ = "Copyright (c) 2018 Dianomic Systems"
+__license__ = "Apache 2.0"
+__version__ = "${VERSION}"
+
+TEMPLATE_NAME = "template.json"
+SENSOR_VALUE = 20
+
+
+def _remove_data_file(file_path=None):
+    if os.path.exists(file_path):
+        os.remove(file_path)
+
+
+def _remove_directories(dir_path=None):
```

Comment:
Move  __remove_directories_ and __remove_data_file_ def to conftest, so that others can re-use it